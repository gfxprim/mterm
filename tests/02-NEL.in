5 5
a\n
b\n
c\n
d\n
e
\eE\eE\eE
