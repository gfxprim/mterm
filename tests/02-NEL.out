 -----
|d    |
|e    |
|     |
|     |
|     |
 -----
size 5x5 cursor 4x0
